test/packages/parallel/terraform_local/data_stream/outputs/_dev/deploy/tf/main.tf

```diff
@@ -0,0 +1,13 @@
+variable "TEST_RUN_ID" {
+  default = "detached"
+}
+
+resource "local_file" "log" {
+  source          = "./files/example.log"
+  filename        = "/tmp/service_logs/${var.TEST_RUN_ID}.log"
+  file_permission = "0777"
+}
+
+output "filename" {
+    value = local_file.log.filename 
```

Comment:
In this data stream , a Terraform output is set and it is used to configure the policy in the system configuration file.